other/management/commands/push-notify.py

```diff
@@ -98,6 +108,10 @@ def handle(self, *args, **options):
                             "data": {
                                 "primaryKey": 1
                             }
+                        },
+                        "data": {
+                            "type": notification_type,
+                            "id": notification_id
```

Comment:
Should this be outside, or inside notification? Please check once.

Reply:
In the [firebase docs](https://firebase.google.com/docs/cloud-messaging/concept-options#notification-messages-with-optional-data-payload) the data is outside the notification.